### PR TITLE
Create-Account page added

### LINKED ISSUE #7: Register Account - Create Account
Create the final component for the Register Account screen

Wait on Aligary for password component

Mobile View:
image

Desktop View:
Blow up and Center

### PR DESCRIPTION
**Pull request by RedPenguin88**
Closes #7.
### Motivation and Context
This PR adds a new "Create Account" page to be used when the user is registering a new account. Specifically it should be one of three pages for use in #21, which will keep track of all the fields and create the user.

For this page, the user simply types in their password, then confirms it in a second field, then can click the "Create Account" button towards the bottom. The page is as follows:

create_account

### Types of Changes
- [x] Add a path '/create-account' in client/src/main.tsx for viewing purposes
- [x] Change MUI components to custom components by @Aligary
- [x] Top bar is resultant from a transparent Appbar from MUI
- [x] Clicking the button currently does nothing, but the `handleClick` function is there for use in #21
### Potential Updates
These updates may be suitable for future PRs.

- [ ] Add some mechanism for checking that the passwords in both fields match, alerting the user if they do not
- [ ] The button at the bottom should be disabled/greyed out until both passwords match
- [ ] The "Back Button" at the top left should navigate to the previous page (to be done in #21)

### CODE REVIEW

**Review by Linkaloo — APPROVED** [review]
As with the other PR related to the account registration page, this wont actually be its own page just a component for the actual register account page. But it is fine using it like this for now

File: client/src/pages/CreateAccount/CreateAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    // For now, it simply prints the values
+    console.log(password);
+    console.log(confirmPassword);
+  };
+
+  return (
+    <div className={styles['Create Account']}>
+      <AppBar
+        position="static"
+        sx={{ background: 'transparent', boxShadow: 'none', color: 'black' }}
```
Comment (Aligary) [inline client/src/pages/CreateAccount/CreateAccount.tsx:22]: You should have the styling in the .scss file and import it from there. 

I see that you are styling an MUI component, Jordan shared this with us that shows how to do so: https://medium.com/@savindiwaduge7/styling-with-mui5-sass-in-a-design-system-bb617b920d13
Although i dont think it makes a huge difference.

File: client/src/pages/CreateAccount/CreateAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      >
+        <Toolbar>
+          <IconButton edge="start" color="inherit" aria-label="back">
+            <ArrowBackIos />
+          </IconButton>
+          <Typography
+            variant="h6"
+            component="div"
+            sx={{ display: 'flex', flexGrow: 1, justifyContent: 'center' }}
```
Comment (Aligary) [inline client/src/pages/CreateAccount/CreateAccount.tsx:31]: same thing here

File: client/src/pages/CreateAccount/CreateAccount.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          >
+            Create Account
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <div style={{ padding: '20px' }}>
```
Comment (Aligary) [inline client/src/pages/CreateAccount/CreateAccount.tsx:37]: and here

**Review by Aligary — CHANGES_REQUESTED** [review]
I believe josh mentioned that these should ultimately be a component but it's fine to be a page for now. Other than that, just had minor comments. Good job!

**Review by morenj — APPROVED** [review]
Beautifully done! I was able to clone the fork, install dependencies, and run the dev environment on my end. Everything worked perfectly and I was able to test the basic functions of the create account fields. The code looks structured, easy to follow, and well organized.

Personally I like styling on the same file, but I suppose it would be best to be consistent across the board.

File: client/src/pages/CreateAccount/CreateAccount.tsx (hunk 1 above)
Comment (RedPenguin88) [reply client/src/pages/CreateAccount/CreateAccount.tsx]: I've moved the styling to the separate `CreateAccount.module.scss` file. For the AppBar component, I had to add the `!important` rule for the styling to work properly. If there are better ways to fix this, let me know!

**Review by Aligary — APPROVED** [review]

**Pull request merged**